test/test_utils.hpp

```diff
@@ -311,8 +315,13 @@ void input_tester<CharOut>::finish()
     }
     if (_expected != _result)
     {
+#ifndef __CUDACC__
         _test_failure( "\n expected: \"", strf::cv(_expected), '\"'
                      , "\n obtained: \"", strf::cv(_result), "\"\n" );
+#else
+        _test_failure( "\n expected: \"", _expected, '\"'
+                     , "\n obtained: \"", _result, "\"\n" );
+#endif
```

Comment:
Now that `input_types/cv_string.hpp` is adapted, maybe this can be reverted.

Reply:
Yes.